db/migrate/20221227064824_set_default_to_sensitive_contents.rb

```diff
@@ -0,0 +1,7 @@
+class SetDefaultToSensitiveContents < ActiveRecord::Migration[6.1]
+  def change
+    change_column :posts, :oc, :boolean, :default => false
+    change_column :posts, :spoiler, :boolean, :default => false
+    change_column :posts, :nsfw, :boolean, :default => false
+  end
+end
```

Comment:
you have this file as well

Reply:
Delete the files that are not required. Please clean up code or files when they are not required. Otherwise this will cause issues for others in the team.

Reply:
ok magesh

Reply:
Migrations file errors are corrected now.